### PR TITLE
Юферев -- Лабораторная работа #1

### PR DESCRIPTION
**Pull request by YuferevKirill**

### CODE REVIEW

**Comment by YuferevKirill** [comment]
Hi, @Nikita-Mitrokhin, @mkildishev, @ksenya-sizova! Please, can you review my code?

File: code/stack/src/main/java/ru/unn/agile/stack/model/Stack.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import java.util.List;
+
+public class Stack<T> {
+    private List<T> list;
+
+    public Stack() {
+        list = new ArrayList<>();
+    }
+
+    public boolean isStackCreated() {
```
Comment (mkildishev) [inline code/stack/src/main/java/ru/unn/agile/stack/model/Stack.java]: Имхо, сомнительная необходимость метода.

Comment (YuferevKirill) [reply code/stack/src/main/java/ru/unn/agile/stack/model/Stack.java]: Fixed)

File: code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    @Test(expected = Test.None.class)
+    public void canPushIntegerToStack() {
+        Stack<Integer> stack = new Stack<>();
+        int valueToAdd = 5;
+
+        stack.push(valueToAdd);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void canThrowWhenPushNullElementToSTack() {
```
Comment (ksenya-sizova) [inline code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java]: Please, change STack to Stack

File: settings.gradle [hunk 3]
```diff
@@ -41,3 +41,7 @@ include "code/korniakov-kirill-lab3/View"
 project(":code/korniakov-kirill-lab3/View").name = "korniakov-kirill-lab3-View"
 include "code/korniakov-kirill-lab3/Infrastructure"
 project(":code/korniakov-kirill-lab3/Infrastructure").name = "korniakov-kirill-lab3-Infrastructure"
+
+// Yuferev Kirill, lab1
+include "code/stack"
+project(":code/stack").name = "stack-Model"
```
Comment (ksenya-sizova) [inline settings.gradle]: There is "No newline at end of file" message. Please fix it.

File: code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+        stack.peek();
+    }
+
+    @Test
+    public void isStackWithoutItemsEmpty() {
+        Stack<Integer> stack = new Stack<>();
+
+        assertTrue(stack.isEmpty());
+    }
```
Comment (ksenya-sizova) [inline code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java]: I think you can add one more test for `isEmpty()` method. For example, you can check that `isEmpty() == false` when there are elements in stack.

Comment (YuferevKirill) [reply code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java]: fixed

File: code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java (hunk 2 above)
Comment (YuferevKirill) [reply code/stack/src/test/java/ru/unn/agile/stack/Model/StackTest.java]: fixed

File: settings.gradle (hunk 3 above)
Comment (YuferevKirill) [reply settings.gradle]: it's not actual anymore)

**Comment by mkildishev** [comment]
👍

**Comment by ksenya-sizova** [comment]
:+1:

**Pull request merged**